Sources/Function.swift

```diff
@@ -109,3 +108,11 @@ func valueString<T>(_ target: T, debug: Bool) -> String {
         preconditionFailure("Not supported type")
     }
 }
+
+func extractKeyValues(from dictionary: Any) -> [(Any, Any)] {
+    Mirror(reflecting: dictionary).children.map {
+        let key = Mirror(reflecting: $0.value).children.first!.value
```

Comment:
強制アンラップでよかです？？

Reply:
細かいですが、
これも共通化したいおのこありけり
`Mirror(reflecting: $0.value).children.`

Reply:
> It's better to use this part in common.

I did it! ( 31db182 )
But don't extract code `.children`, because I can't think of better naming for local variable.